presto-main/src/main/java/com/facebook/presto/execution/UseTask.java

```diff
@@ -68,16 +74,18 @@ private void checkCatalogAndSessionPresent(Use statement, Session session)
         }
     }
 
-    private void checkAndSetCatalog(Use statement, Metadata metadata, QueryStateMachine stateMachine, Session session)
+    private void checkAndSetCatalog(Use statement, Metadata metadata, QueryStateMachine stateMachine, Session session, AccessControl accessControl)
     {
         if (statement.getCatalog().isPresent()) {
-            String catalog = statement.getCatalog().get().getValueLowerCase();
+            String catalog = statement.getCatalog()
+                    .map(Identifier::getValueLowerCase)
```

Comment:
Why is this needed? `statement.getCatalog().isPresent()` is already checked above, so catalog is there

Reply:
changed code accordingly